app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -236,21 +187,68 @@ class CustomChatInput extends ConsumerWidget {
     );
   }
 
+  Widget replyBuilder(BuildContext context, WidgetRef ref, Widget? child) {
+    final roomNotifier = ref.watch(chatRoomProvider.notifier);
+    final authorId = roomNotifier.repliedToMessage!.author.id;
+    final replyProfile = ref.watch(memberProfileProvider(authorId));
+    final inputNotifier = ref.read(chatInputProvider.notifier);
+    return Row(
+      children: [
+        replyProfile.when(
+          data: (profile) {
+            return ActerAvatar(
+              mode: DisplayMode.User,
+              uniqueId: authorId,
+              displayName: profile.displayName ?? authorId,
+              avatar: profile.getAvatarImage(),
+              size: profile.hasAvatar() ? 12 : 24,
+            );
+          },
+          error: (e, st) => Text(
```

Comment:
So as I looked upon these avatars render methods, we want to return it to fallback in case of errors and log the error event instead of making it appear on the UI. Use ```ActerAvatar()``` too here and print the error log. Same goes for all the places where  avatar ```AsyncValue``` is being handled.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/855/commits/cc1ed83aa45d28c1d26f89ea6e3e0d0ce4e52503